recipes/symengine/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
```

Comment:
@patricksnape, can you please enlighten us on the quote choices made with `%CMAKE_GENERATOR%`?